bin/compile

```diff
@@ -38,8 +38,12 @@ echo "-----> backing up to s3 ... "
 #compress actual code
 tar -cvzPf /tmp/backup.tgz $BUILD_DIR > /dev/null
```

Comment:
you're using the verbose output (v flag), but then just dumping stdout it to /dev/null.